### PR TITLE
Add a timer inhibit if the daemon took a long time to startup

### PR DESCRIPTION
**Pull request by hughsie**
This also allows us to decrease the value of the idle timer.

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [X] Feature
- [ ] Documentation

### CODE REVIEW

File: docs/fwupd.conf.md [hunk 1]
```diff
@@ -61,6 +61,11 @@ The `[fwupd]` section can contain the following parameters:
 
   **NOTE:** some plugins might inhibit the auto-shutdown, for instance thunderbolt.
 
+**IdleInhibitSlowStartup={{IdleInhibitSlowStartup}}**
```
Comment (hughsie) [inline docs/fwupd.conf.md:64]: @superm1 better names welcome for this.

Comment (superm1) [reply docs/fwupd.conf.md:64]: How about using the word threshold

Comment (hughsie) [reply docs/fwupd.conf.md:64]: So `IdleInhibitStartupThreshold` perhaps?

Comment (superm1) [reply docs/fwupd.conf.md:64]: Yes

Comment (hughsie) [reply docs/fwupd.conf.md]: Done.

**Review by superm1 — APPROVED** [review]

**Pull request merged**